src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.html

```diff
@@ -15,7 +15,7 @@
        target="_blank"
        i18n>documentation</a>
     <a ngbDropdownItem
-       href="/docs"
+       href="#/api-docs"
```

Comment:
I don't think we need the hash here but you have to change this with a routerLink (you may check the navigation bar for reference.
```suggestion
       routerLink="/api-docs"
```

Reply:
Thanks @epuertat . Done